vortex-scan/src/row_mask.rs

```diff
@@ -25,9 +24,7 @@ pub struct RowMask {
 #[cfg(test)]
 impl PartialEq for RowMask {
     fn eq(&self, other: &Self) -> bool {
-        self.begin == other.begin
-            && self.end == other.end
-            && self.mask.to_boolean_buffer().unwrap() == other.mask.to_boolean_buffer().unwrap()
+        self.begin == other.begin && self.end == other.end && self.mask == other.mask
```

Comment:
if mask derives PartialEq then we can just derive it on RowMask, this was behind cfg(test) since getting boolean_buffer might have been expensive